### ISSUE TITLE
width configuration option for list pickers to make 'grid-pickers'

### ISSUE THREAD

**Issue by Hendy** [uComponents/nuPickers#171]
![image](https://cloud.githubusercontent.com/assets/77045/25913676/7a56c966-35b3-11e7-8e5d-53ef359f210f.png)

**Comment by greystate** [comment]
Okay - had a quick dabble with the basic layout here: http://dabblet.com/gist/5aa2e32044cfef3c8a9084b7707a536e

Try to add more items to either "column" and see how it _stacks up_ ... is that how it's supposed to work?

(Could probably also try using CSS Grid just for fun, though I feel like flexbox is the better choice here...)

/Chriztian

**Comment by uniquelau** [comment]
That's fun! You could even allow the editor to configure how many columns the picker had very easily.

It would be super nice if it was possible to make 'inline' selects. 
When an item was tapped or clicked a border could appear showing it was selected.

The "Selected items" column on the right could then be optional, depending on the size of data-set / expected number of selected items.

**Comment by greystate** [comment]
Yes - e.g. if you don't need to sort the picked items you wouldn't really need the right column either (unless amount of items to pick from was "huge", as they say :-).

**Comment by Hendy** [comment]
The list on the right can also indicate the min / max number of items to pick - but there could be another config option to enable / disable this list
